tasks/TASK_INDEX.md

```
@@ -943,6 +948,17 @@ created: 2026-02-03
 title: Configure Storybook build and deployment
 ## index_entry_end
 
+## index_entry_begin
```

markdownlint-cli2 (0.20.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)